src/DIRAC/MonitoringSystem/Client/MonitoringClient.py

```diff
@@ -17,7 +17,7 @@ class MonitoringClient(Client):
     def __init__(self, **kwargs):
         """Simple constructor"""
 
-        super(MonitoringClient, self).__init__(**kwargs)
+        super().__init__(**kwargs)
```

Comment:
This, for a reason I had no time to investigate, does not work. Restore it as before.